SIDFactoryII/source/runtime/editor/converters/utils/misc.cpp

```diff
@@ -17,11 +17,10 @@ namespace Converter
 	{
 		// Colors only work in Visual Studio's console
 		if (!m_IgnoreColors)
-			std::string m_Zebra[2] =
-			{
-				"\033[0m",				// Reset (default gray)
-				"\033[1m\033[37m"		// White
-			};
+		{
```

Comment:
I changed this because of a warning, can you check if this is correct @TheRealLaxity ?

Reply:
@TheRealLaxity Says it is :)